Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -24,6 +26,8 @@ public sealed class QuickPulseTelemetryModule : ITelemetryModule, IDisposable
     {
         private const int MaxSampleStorageSize = 10;
 
+        private const int TopCpuCount = 5;
```

Comment:
Will customers ever be able to configure a different top count they want to see?